presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/TestGeometryMemorySizeInBytes.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial;
+
+import com.esri.core.geometry.ogc.OGCGeometry;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.geospatial.GeometryUtils.getEstimatedMemorySizeInBytes;
+import static org.testng.Assert.assertEquals;
+
+public class TestGeometryMemorySizeInBytes
```

Comment:
These tests are brittle. The layout of a class is an internal implementation detail of the JVM and can change across versions or even VM settings (e.g, if compressed oops are enabled, which happens automatically for heaps larger than 30 gb)

Reply:
These a good points. I wanted to do some sanity checks to make sure that getEstimatedMemorySizeInBytes works (doesn't fail) for all geometry types. Will it be better if I change the test to verify that size > 0 ? Or do you have a better suggestion?